src/renderer/deviceMessages.ts

```diff
@@ -1,13 +1,34 @@
 import { BackendRemote } from './backend-com'
 
-export async function updateDeviceChats(accountId: number) {
+export async function updateDeviceChats(
+  accountId: number,
+  skipCurrentChangelog: boolean = false
+) {
+  const UpdateMessage = async (
```

Comment:
Function names are usually written in lowerCamelCase and classes are in UpperCamelCase, so I'd change this to `updateMessage` but since it is only used once here, why a function at all?

Reply:
you might have multiple device messages you want to send, also to separate the if statement from the content, otherwise you'd need to repeat the label.

Reply:
other option could be an array of device messages, but I think that would be overkill.

Reply:
anyways not important enough to discuss